tasks.py

```diff
@@ -31,11 +33,10 @@ def song_links():
 
 @app.task
 def song_lyrics(url, title):
-    page = requests.get(url, headers={'user-agent': USER_AGENT})
-    content = page.text.replace('<br>', '').encode('utf-8')
-    parsed_content = BeautifulSoup(content, 'html.parser')
-    ringtone = parsed_content.find('div', {'class': 'ringtone'})
-    lyrics = ringtone.find_next_sibling('div').text
+    page = requests.get(url, headers={'user-agent': 'Mozilla/5.0 (compatible; Googlebot/2.1; +http://www.google.com/bot.html)'})
+    # have to mimic google bot here
+    parsed_content = BeautifulSoup(page.content, 'html.parser')
+    lyrics = parsed_content.find('p', {'id':'songLyricsDiv'}).text
     filename = os.path.join(OUTPUT_DIR, '{}.txt'.format(title))
```

Comment:
There is a small problem with how this is constructed with the titles they have (not sure if it was a problem before). Some of the titles contain '/' characters like `Blank Space (Guitar/Vocal) (Voice Memo)` and that creates and error. We should replace those with something. Maybe just a space.